src/mds/agg_mds/adapters.py

```diff
@@ -914,7 +916,7 @@ def getRemoteDataAsJson(self, **kwargs) -> Dict:
                     url += f"&{filters}"
                 if field_name is not None and field_value is not None:
                     url += f"&{guid_type}.{field_name}={field_value}"
-                response = httpx.get(url)
+                response = httpx.get(url, timeout=60)
```

Comment:
could you update the PR description, `Updating max timout to 60 seconds for getting AggMDS (MIDRC timeout issue)` is not generic and external-facing